### ISSUE TITLE
Node.js Wrapper?

### ISSUE THREAD

**Issue by ohenepee** [USCiLab/cereal#429]
Could it be added in the roadmap to build a native node.js addon/package of this wonderful serialization library?

**Comment by lukeramsden** [comment]
JS has this built in with JSON: **J**ava**S**cript **O**bject **N**otation. Just run `JSON.stringify(object)` and then save the string to a file.

**Comment by ohenepee** [comment]
Yeah, but performance is nothing compared to `cereal` and since NodeJS allows C++ wrappers... was thinking...

**Comment by itzhaolin** [comment]
we need npm packages like c++ cereal!!!
